payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetAddPaymentMethodFragment.kt

```diff
@@ -30,6 +29,12 @@ internal class PaymentSheetAddPaymentMethodFragment(
         viewModelFactory
     }
 
+    private lateinit var viewBinding: FragmentPaymentsheetAddPaymentMethodBinding
+
+    // Holds a reference to the last selected payment method while checking out with Google Pay.
+    // If Google Pay is cancelled or fails, it will be set again as the selected payment method.
+    private var lastSelectedPaymentMethod: PaymentSelection? = null
```

Comment:
is it posible to hold this in ViewModel?

Reply:
Done